api/src/main/java/org/openmrs/module/aijar/tasks/GenerateUniqueIdentifierCodeTask.java

```diff
@@ -0,0 +1,235 @@
+package org.openmrs.module.aijar.tasks;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.context.Context;
+import org.openmrs.scheduler.tasks.AbstractTask;
+
+public class GenerateUniqueIdentifierCodeTask extends AbstractTask {
+    private Log log = LogFactory.getLog(this.getClass());
+    @Override
+    public void execute() {
+        log.info("method executing started");
+        generateUICForPatientsWithout();
```

Comment:
Make this job delete or disable it self after it has completed running. I believe its a one time Job

Reply:
No it's not a one time job since we can have new patient registered  into a system